pallets/stateful-storage/src/lib.rs

```diff
@@ -325,11 +320,12 @@ pub mod pallet {
 			msa_id: MessageSourceId,
 			schema_id: SchemaId,
 		) -> Page<T::MaxItemizedPageSizeBytes> {
-			let storage_key = &schema_id.encode()[..];
-			let keys = vec![storage_key.to_vec()];
-			let page_response =
-				StatefulChildTree::try_read::<Page<T::MaxItemizedPageSizeBytes>>(&msa_id, &keys)
-					.map_or_else(|_| Page::default(), |page| page.unwrap_or_default());
+			let key: ItemizedFullKey = (schema_id,);
+			let page_response = StatefulChildTree::<T::Hasher>::try_read::<
```

Comment:
Can we call it like this?
StatefulChildTree::<T::Hasher>::try_read::<_, ...>(&msa_id, &key). unwrap_or_default()

Reply:
No... the return value is an `Option` wrapped in a `Result`. `unwrap_or_default()` on the Result may still yield a `None` value.